doc/developers/contributing.rst

```diff
@@ -444,15 +444,17 @@ to the documentation, e.g.::
 
     pip install --editable ..
 
-To generate the full web site, including the example gallery (this might take a
-while)::
+To generate the full web site, including the example gallery::
 
     make html
 
-Or, if you'd rather quickly generate the documentation without the example
-gallery::
-
-    make html-noplot
+Generating the example gallery will run all our examples which takes a
+while. To save some time, you can use:
+    - ``make html-noplot``: this will generate the documentation without the
+      example gallery. This is useful when changing a docstring for example.
+    - ``PATTERN=your_regex_goes_here make html-pattern``: only the examples
```

Comment:
PATTERN -> EXAMPLES ?

Reply:
Hmmm OK this is a regex so I am not sure EXAMPLES is a great name. Maybe EXAMPLES_PATTERN although I liked that the target suffix and the environment variable matched. Full disclosure: I copied the names from mne-python ...